src/qililab/qprogram/quantum_machines_compiler.py

```diff
@@ -117,7 +117,6 @@ class QuantumMachinesCompiler:
 
     FREQUENCY_COEFF = 1
     PHASE_COEFF = 2 * np.pi
-    VOLTAGE_COEFF = 2
```

Comment:
Why did we remove the VOLTAGE_COEFF? Did their ranges change?

Reply:
Yes, QM changed the way they use their amplitudes (both in waveforms and in the gains inside a play/measure). I tested this in an oscilloscope, I can send you the details through slack (images and the config file).

Reply:
Yes please send the details. Also we should document this in the changelog.